proposal/support-selector&.md

```diff
@@ -0,0 +1,31 @@
+# Support `selector&`: Draft 1
```

Comment:
I'd like the title to be a little clearer than this. "Support" is a bit of a filler word, and we try to avoid putting punctuation in proposal names. Maybe "Prefixed Parent Selector"?

Reply:
Alright, and I completely agree!